demo/examples/toggle-box.jsx

```diff
@@ -87,7 +87,7 @@ var ToggleBox = React.createClass({
     return (
       <div className="flex-box flex-column flex-1 align-items-center">
         <div>
-          <select rvalue={this.state.effect} onChange={this.whenSelectChanged}>{this.renderEffects()}</select>
```

Comment:
It should probably just be `value`.

On Fri, Aug 5, 2016 at 10:56 AM, Anthony Barone <notifications@github.com>
wrote:

> In demo/examples/toggle-box.jsx
> <https://github.com/twitter-fabric/velocity-react/pull/118#discussion_r73705754>
> :
>
> > @@ -87,7 +87,7 @@ var ToggleBox = React.createClass({
> >      return (
> >        <div className="flex-box flex-column flex-1 align-items-center">
> >          <div>
> > -          <select rvalue={this.state.effect} onChange={this.whenSelectChanged}>{this.renderEffects()}</select>
>
> Wondering what the intent of this property was, can't find information on
> it anywhere. Plus it through an "Unknown property" warning (same thing down
> below in trigger-box).
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/twitter-fabric/velocity-react/pull/118/files/74664beda670a05df4be99b0f5a185b0e8bbdc89#r73705754>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAP65QXPOjmp9spInqB8QIsfw0P_2fi8ks5qc08rgaJpZM4JdwiN>
> .
>